data/json/items/book/cooking.json

```diff
@@ -516,5 +552,77 @@
     "max_level": 4,
     "intelligence": 7,
     "copy-from": "cookbook_liverforkids"
+  },
+  {
+    "id": "winemaking_beginner",
+    "type": "BOOK",
+    "name": { "str": "Winemaking for Beginners", "str_pl": "copies of Winemaking for Beginners" },
+    "description": "Winemaking, is an art form.  This book teaches you how to crayon within the lines.",
+    "weight": "454 g",
+    "volume": "1 L",
+    "price": 4000,
+    "price_postapoc": 31500,
+    "bashing": 2,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "red",
+    "skill": "cooking",
+    "required_level": 1,
+    "max_level": 2,
+    "time": "12 m"
+  },
+  {
+    "id": "manual_canning",
+    "type": "BOOK",
+    "name": { "str": "Canning Pot Instructions", "str_pl": "copies of Canning Pot Instructions" },
+    "description": "Exciting recipes and restaurant reviews.  Full of handy tips about cooking.",
```

Comment:
```suggestion
    "description": "This entry-level guide to home canning talks about the importance of acidity and food contents in preservation, and how to avoid contaminants in your canning batches.",
```